test/unit/model/test_more_train.py

```diff
@@ -53,49 +54,62 @@
     model_trainer_ref = TestData.load("model_trainer_reference.pickle")
 
 
+model, model_file = serializer.load(123, TestData.TRAINED_MODELS_FOLDER)
+model_path = TestData.TRAINED_MODELS_FOLDER / "<pid>/20191119120000/model.bin"
+
+
+def get_serializer_mock():
+    serializer_mock = MagicMock()
+    serializer_mock.load.return_value = model, model_path
+
+
+# @patch("stf.model.serializer.creator.XGBModelSerializer", get_serializer_mock())
```

Comment:
Is this commented out code functional or can we remove it?

Reply:
Good point. I have already removed this on a newer branch.

Reply:
To be sure, I removed the comment on this branch as well.